documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -21,6 +21,11 @@ include::snippet_ova_tech_preview.adoc[]
 Migration using {osp} source providers only supports VMs that use only Cinder volumes.
 ====
 
+[NOTE]
+====
+You can create `Provider` manifests for future use if the virtual machines that they include currently lack storage.
```

Comment:
I still don't understand what this means.. that's related to the empty mappings we talked about before, right?
in that case, I'd suggest to add to `network: <3>` below a note that says something like: network mapping should be specified also if the vms to be migrated don't have networks, the mapping can be empty in this case.

btw, I'm looking at `network: <3>` below and seems we need to change the current description of this item ("You can add multiple network mappings") - the plan can point to a single network (or storage) mapping

Reply:
@ahadas  about changing "You can add multiple network mappings" Are you saying we allow one network map per plan? If so, do I need to state this for storage below (no callout now).

Reply:
@RichardHoch correct. the network map can map multiple source networks to destination networks